### DISCUSSION TITLE
请求在“图片链接”中添加“使用图片原链接“选项

### DISCUSSION THREAD

**Discussion by s0ren99** [Chilfish/Weibo-archiver#53] [功能请求]
如标题。

用Header Editor之类的插件修改请求头后，就可以直接浏览了，网上教程很多。

之前用另一个插件备份的很多html, 通过这个办法修改之后图片都是瞬间显示的。

默认的CDN图片加载不动...

**Comment by Chilfish** [comment]
谢谢提醒，忘了还可以这么做❤️现在可以在设置里选上使用微博原图了

不过除了微博图片以外 b站也有防盗链，所以也要添加B站的匹配规则 `.+(sinaimg.cn|hdslb.com)`
